selfdrive/car/hyundai/values.py

```diff
@@ -20,7 +20,7 @@ def __init__(self, CP):
     self.STEER_DRIVER_FACTOR = 1
     self.STEER_THRESHOLD = 150
 
-    if CP.carFingerprint in HDA2_CAR:
+    if CP.carFingerprint in HDA2_CAR and CP.carFingerprint not in (CAR.GENESIS_GV70):
```

Comment:
```suggestion
    if CP.carFingerprint in HDA2_CAR and CP.carFingerprint not in (CAR.GENESIS_GV70,):
```

Assuming the torque is the same though, let's stick with what we found acceptable in the EV6

Reply:
This might be different with the lfa command though, I can just check out a route when you're ready for review to find the max lateral acceleration

Reply:
Did you change panda? It might have been blocking messages

Reply:
@sshane should I keep STEER_DRIVER_ALLOWANCE, STEER_DRIVER_MULTIPLIER, STEER_THRESHOLD at the EV6 values, or the default hyundai? Or does this need to be calculated later?

Reply:
We can check later, but try default EV6 values